### ISSUE TITLE
Accessibility: Audit Editor

### ISSUE THREAD

**Issue by khaykov** [wordpress-mobile/WordPress-Android#7288]
Audit Editor, on both Aztec and wpandroid sides, to make sure that all the functionality is accessible with TalkBack and Switch Access.

- All the clickable elements should be marked as so.
- All nonactionable and decorative elements should be non-focusable.
- Relevant content should be grouped together.
- All important elements should be described through content description.
